events/job_chancellor.txt

```diff
@@ -375,7 +375,9 @@ character_event = {
 	}
 	
 	mean_time_to_happen = {
-		months = 360
+		# Warcraft
+		months = 180
```

Comment:
Gamerule on this, pls
Normal/faster/slower

Reply:
Added, but mod "normal" speed is 75% of vanilla normal speed :3